Sources/MapboxCoreNavigation/EHorizon/RoadObjectsStore.swift

```diff
@@ -56,6 +56,36 @@ public final class RoadObjectsStore {
         return native.getRoadObjectIdsByEdgeIds(forEdgeIds: edgeIdentifiers.map(NSNumber.init))
     }
 
+    /**
+     Adds a road object to be tracked in the electronic horizon. In case if an object with such identifier already exists, updates it.
+     NB: road objects obtained from route alerts cannot be added via this API.
+
+     - parameter roadObject: Custom road object, acquired from `RoadObjectMathcer`.
+     */
+    public func addCustomRoadObject(_ roadObject: RoadObject) {
```

Comment:
Looks like "user-defined" won't be accurate at all times, as these objects may be manipulated by the SDK client but not the end-user. This is something not provided out-of-the-box, not necessarily user-initiated. I see Maps are using the "custom" term for styles and events which is semantically similar to the context of road objects.

Reply:
`add` looks OK without such specification, however both `remove` and `removeAll` will only work for custom objects.

Reply:
> Looks like "user-defined" won't be accurate at all times, as these objects may be manipulated by the SDK client but not the end-user.

I should’ve clarified: “user-defined” in the API would refer to the developer, not the end user, as in `userInfo` in notifications.

Reply:
Yeah, we can interpret user as developer, but Cocoa APIs I've checked mostly imply end-user (user defaults - user settings, UserActivity, UserLocation, etc.). I'd say "custom" is a more descriptive term, but that I guess is a matter of taste.

Reply:
It may be that only “user-defined” is used this way, but not “user” in general. Xcode also has a “user-defined runtime attributes” feature that is actually defined by developers.